pnmtopng: The -phys option no longer exists.

From a fresh git clone:
```
make all
giftopnm images/caution.gif | pnmtopng -phys 11811 11811 1 | convert - images/caution.eps
pnmtopng: The -phys option no longer exists.  Use -size instead.
convert: no decode delegate for this image format `' @ error/constitute.c/ReadImage/746.
convert: no images defined `images/caution.eps' @ error/convert.c/ConvertImageCommand/3342.
make: *** [Makefile:175: images/caution.eps] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -172,10 +172,10 @@ images/%.jpg : images/%.gif
 	convert -resize 50% images/$*.gif images/$*.jpg
 
 images/%.eps : images/%.gif
-	giftopnm images/$*.gif | pnmtopng -phys 11811 11811 1 | convert - images/$*.eps
+	giftopnm images/$*.gif | pnmtopng -size "11811 11811 1" | convert - images/$*.eps
 
 images/%.png : images/%.gif
-	giftopnm images/$*.gif | pnmtopng -phys 11811 11811 1 > images/$*.png
+	giftopnm images/$*.gif | pnmtopng -size "11811 11811 1" > images/$*.png
 
 .PHONY: clean
 clean:

